internal/tiltfile/include/include.go

```diff
@@ -11,6 +11,9 @@ import (
 // The main difference is that include() doesn't bind any arguments into the
 // global scope, whereas load() forces you to bind at least one argument into the global
 // scope (i.e., you can't load() a Tilfile for its side-effects).
+//
+// Users should generally be discouraged from using this function. They should use
```

Comment:
is it worth deprecating `include`?

Reply:
probably eventually, tho i think it's good practice to leave it around for a little while before formally deprecating